documentation/Color system.md

```diff
@@ -236,6 +240,8 @@ Used to decorate elements where color does convey a specific meaning in componen
 | `--p-override-loading-z-index`            | `514`                                                                                             |
 | `--p-button-font-weight`                  | `500`                                                                                             |
 | `--p-non-null-content`                    | `''`                                                                                              |
+| `--p-choice-size`                         | `2rem`                                                                                            |
+| `--p-choice-margin`                       | `0.1rem`                                                                                          |
```

Comment:
This seems a little odd to me at first glance. These variables seem like they would only be used within the one component, not globally for other components / layouts. I could be wrong though

Reply:
I think we'll be able to delete this one in the end. This margin doesn't exist today which is why I added it here. It could simply be added via the classname as well. I'll do that.

Reply:
Yes, these variables are temporary.